src/core/client_channel/subchannel.h

```diff
@@ -88,13 +88,15 @@ class ConnectedSubchannel : public RefCounted<ConnectedSubchannel> {
   virtual void Ping(grpc_closure* on_initiate, grpc_closure* on_ack) = 0;
 
   virtual channelz::SubchannelNode* channelz_node() const = 0;
-  std::string security_level_;
+
+  absl::string_view GetSecurityLevel() { return security_level_; }
```

Comment:
This can just be named `security_level()`, as per style guide rules for accessor methods:

https://google.github.io/styleguide/cppguide.html#Function_Names